InputTime/index.jsx

```diff
@@ -7,7 +8,7 @@ const leftPadTimeUnit = timeUnit => (timeUnit < 10 ? `0${timeUnit}` : timeUnit);
 /* eslint-disable react/prop-types */
 
 const renderAmPm = ({ value, onChange, submitting }) =>
-  <select
+  (<select
```

Comment:
I don't think we should convert all of these - it's extra characters in the commits and the code.